sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+# Check WRT BuiltInCall
+
+##@@
+
+## Operators
+
+sparql:plus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:subtract rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:multiple rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:divide rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:unary-minus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:unary-plus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+# Section: Functional Forms
+
+sparql:bound rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-bound> ;
+    .
+
+sparql:if rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-if> ;
+    .
+
+sparql:coalesce rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-coalesce> ;
+    .
+
+sparql:filter-exists rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-filter-exists> ;
+    .
+
+sparql:logical-or rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-logical-or> ;
+    .
+
+sparql:logical-and rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-logical-and> ;
+    .
+
+sparql:in rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-in> ;
+    .
+
+sparql:not-in rdf:type sparql:FunctionalForm ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-not-in> ;
+    .
+
+# Section: Functions on RDF Terms
+
+sparql:sameTerm rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-sameTerm> ;
+    .
+
+sparql:sameValue rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-sameValue> ;
+    .
+
+# Deprecated: RDFterm-equal
+
+sparql:RDFterm-equal rdf:type sparql:Function ;
+    dc:description "Deprecated" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql11-query/#func-RDFterm-equal> ;
+    .
+
+sparql:isIRI rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-isIRI> ;
+    .
+
+sparql:isBlank rdf:type sparql:Function ;
+    dc:description "" ;
```

Comment:
This function returns true if the provided term is a blank node, and false otherwise.

Reply:
```suggestion
    dc:description "This function returns true if the provided term is a blank node, and false otherwise." ;
```